### PR TITLE
25pc non uk

### PR DESCRIPTION
**Pull request by commuterjoy**
/cc @ifyio

### CODE REVIEW

**Comment by commuterjoy** [comment]
Fixed a few things in 359287c438078db4b0d784d1cff0a248146286b7

File: server/init.js [hunk 1]
```diff
@@ -81,6 +83,12 @@ function isB2B(req, res, next) {
 	if(req.get('x-b2b-type') === 'licence' && res.locals.flags.signupFormB2B) {
 		signupB2B(req, res, next);
 	} else {
-		signup(req, res, next);
+		if ((req.get('Country-Code') === 'GB' || Math.random() < 0.25)) {
```
Comment (lc512k) [inline server/init.js]: all GB users and 25% of everyone else go to the old form?

**Comment by lc512k** [comment]
smoke.js fails if we redirect GB visits

File: server/init.js (hunk 1 above)
Comment (commuterjoy) [inline server/init.js]: actually, yes, we should try to keep 10% of UK users on the current form.

Comment (ifyio) [inline server/init.js]: The concern is that they would loose out on direct debit as that is UK only. But I think that is really only an issue that is specific to the A/B test that they wanted to do so we should be able to keep UK users on the form, if we are not doing an A/B test. Will have a chat with Purvi

**Pull request merged**